### ISSUE TITLE
[Windows] Crash on login after syncing egl installed games

### ISSUE THREAD

**Issue by TSRBerry** [suchmememanyskill/Legendary-GUI#2]
Hello,

Yesterday I tried to setup Duplicate as I thought the automatic shortcuts for Steam would be really useful.

But I had some trouble to get it to detect all my installed games (even after selecting the correct install folder). At first I thought I could solve this by clicking install on every game that I already have installed, but that led to a crash after the 4th time I clicked on an item to "install" it.

So I looked at legendary itself and discovered that I could import all my games from EGL which is really cool. I did that and legendary reports that it imported all my games. But when I try to run Duplicate now, I don't get past the "logging in..." screen and it crashes.

Duplicate Crash log (from Event Viewer):
```
Application: Duplicate.exe
CoreCLR Version: 5.0.1321.56516
.NET Version: 5.0.13
Description: The process was terminated due to an unhandled exception.
Exception Info: Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: {. Path 'Turkey.prereq_info', line 1220, position 20.
   at Newtonsoft.Json.JsonTextReader.ReadStringValue(ReadType readType)
   at Newtonsoft.Json.JsonTextReader.ReadAsString()
   at Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateDictionary(IDictionary dictionary, JsonReader reader, JsonDictionaryContract contract, JsonProperty containerProperty, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value)
   at LegendaryMapperV2.Service.LegendaryGameManager.GetGames()
   at LegendaryGUIv2.ViewModels.MainViewModel..ctor(LegendaryAuth auth, MainWindowViewModel window)
   at LegendaryGUIv2.ViewModels.MainWindowViewModel.SetMainViewModel()
   at LegendaryGUIv2.ViewModels.MainWindowViewModel.OnLogin(LegendaryAuth a)
   at LegendaryMapperV2.Service.LegendaryAuth.<>c__DisplayClass13_0.<AttemptLogin>b__0(LegendaryCommand x)
   at LegendaryMapperV2.Service.LegendaryCommand.<Callback>b__25_0(CommandCallback x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at LegendaryMapperV2.Service.LegendaryCommand.Callback(Terminal t)
   at LegendaryMapperV2.Service.Terminal.TrackProc()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ThreadHelper.ThreadStart()
```

Legendary installed games (some games are still missing):
```ps1
PS D:\Program Files\Duplicate> .\legendary egl-sync
[cli] INFO: Reading EGL game manifests...

Checking for importable games...
Nothing to import.

Checking for exportable games...
Nothing to export.

Checking automatic sync...
PS D:\Program Files\Duplicate> .\legendary list-installed

Installed games:
 * 2064: Read Only Memories (App name: f4e0c1dff48749fa9145c1585699e276 | Version: 030521a_up | Platform: Windows | 0.99 GiB)
 * 20XX (App name: Quail | Version: 1.42.12rc1 | Platform: Windows | 1.54 GiB)
 * >observer_ (App name: Tumeric | Version: 1.0.0d | Platform: Windows | 8.63 GiB)
 * Absolute Drift (App name: 19927295d6e3467887d4e830d8c85963 | Version: Win_202201121639 | Platform: Windows | 0.35 GiB)
 * ABZU (App name: Curry | Version: 1.0 | Platform: Windows | 4.49 GiB)
 * ARK: Survival Evolved (App name: aafc587fbf654758802c8e41e4fb3255 | Version: 342.16.win | Platform: Windows | 125.03 GiB)
 * Borderlands 2 (App name: Dodo | Version: 2.0.1 BN264 CL2464059 | Platform: Windows | 22.23 GiB)
 * Borderlands: The Pre-Sequel (App name: Turkey | Version: 2.0.1 BN133 CL2464059 | Platform: Windows | 13.20 GiB)
 * Darksiders II Deathinitive Edition (App name: Hoki | Version: 1.0 | Platform: Windows | 14.57 GiB)
 * Dauntless (App name: Jackal | Version: 1.9.0.328374 | Platform: Windows | 15.13 GiB)
 * Elite Dangerous (App name: 9c203b6ed35846e8a4a9ff1e314f6593 | Version: 3.8.279702 | Platform: Windows | 19.60 GiB)
 * Farming Simulator 19 (App name: Stellula | Version: 1.7.1.0o Win | Platform: Windows | 14.46 GiB)
 * Faster Than Light (App name: Blowfish | Version: 1.6.12.Windows | Platform: Windows | 0.27 GiB)
 * Fez (App name: Bluebird | Version: 1.12.0 | Platform: Windows | 0.42 GiB)
 * Figment (App name: ed5e8d42b2ac461eb6f9176d163b9515 | Version: v1.4.0_2020-03-29_3ceaeed | Platform: Windows | 0.70 GiB)
 * Football Manager 2020 (App name: bf35243e8c3542e082bc7042e0592183 | Version: 1260-win | Platform: Windows | 5.45 GiB) * For Honor (App name: Sundrop | Version: 1602 | Platform: Windows | 67.20 GiB)
  -> Update available! Installed: 1602, Latest: 1902
 * Frostpunk (App name: b2e00fc16632475782a7730d1bd7cb60 | Version: 0.0.10 | Platform: Windows | 8.98 GiB)
 * Galactic Civilizations III (App name: 5f1c66366b2c4a6496d535d88c56e6cf | Version: 4.21.273666a | Platform: Windows | 14.21 GiB)
 * Godfall (App name: Carp | Version: 4.1.9-prod-stable | Platform: Windows | 48.80 GiB)
 * Grand Theft Auto V (App name: 9d2d0eb64d5c44529cece33fe2a46482 | Version: 1.0.2545.0.1 | Platform: Windows | 105.25 GiB)
 * Hello Neighbor (App name: Ursus | Version: 1.4.1 | Platform: Windows | 5.25 GiB)
 * Hue (App name: Herring | Version: 1.0 | Platform: Windows | 0.45 GiB)
 * Hyper Light Drifter (App name: Parakeet | Version: 1.0.0 | Platform: Windows | 0.85 GiB)
 * Just Cause 4 (App name: Kakopo | Version: JC4_1830880_1830540 | Platform: Windows | 47.06 GiB)
 * Layers of Fear (App name: Cilantro | Version: 1.1.1e | Platform: Windows | 4.23 GiB)
 * Layers of Fear 2 (App name: 473e24df09f843a1acc4aa16b01dae83 | Version: 1.0.0.0 | Platform: Windows | 14.33 GiB)
 * Lifeless Planet: Premier Edition (App name: 860a4feefe2a4f48905ae86c61d5db64 | Version: 1.0.0 | Platform: Windows | 0.92 GiB)
 * Limbo (App name: Hazelnut | Version: 1.0.1 | Platform: Windows | 0.10 GiB)
 * Little Inferno (App name: Clover | Version: 1.0 | Platform: Windows | 0.19 GiB)
 * Neon Abyss (App name: a26f991a5e6c4e9c9572fc200cbea47f | Version: Release_59 | Platform: Windows | 0.94 GiB)
 * Neverwinter (App name: 87be9bd6707748888bce1f79b025c5dd | Version: CL_2020_09_25_09_51 | Platform: Windows | 0.15 GiB)
 * Overcooked (App name: Sage | Version: 1.0 | Platform: Windows | 0.69 GiB)
 * Overcooked! 2 (App name: Potoo | Version: 86 | Platform: Windows | 8.05 GiB)
 * PC Building Simulator (App name: ab277c0995e945d2b2c50c46883627f1 | Version: v1.14.2_65 | Platform: Windows | 21.00 GiB)
 * Pikuniku (App name: 48619f5656da446abccb727b78cda644 | Version: Win64_1 | Platform: Windows | 0.28 GiB)
 * Q.U.B.E. 2 (App name: Auk | Version: 1.2 | Platform: Windows | 3.47 GiB)
 * Rocket League® (App name: Sugar | Version: BC2_Update36+355454 | Platform: Windows | 18.22 GiB)
 * Satisfactory Early Access (App name: CrabEA | Version: 176027.281 | Platform: Windows | 17.61 GiB)
 * Slime Rancher (App name: Corydalis | Version: 1.4.4_Win64_Release_16 | Platform: Windows | 1.14 GiB)
 * Sonic Mania (App name: 818447bb519b46d48d365d5753362796 | Version: 1.07.0035 | Platform: Windows | 0.21 GiB)
 * Spellbreak (App name: Newt | Version: release.15058-7706 | Platform: Windows | 23.12 GiB)
 * Stranded Deep (App name: 02107cba432c4551a027d25d597adc49 | Version: Stable-Build_0.90.11 | Platform: Windows | 2.35 GiB)
 * Subnautica (App name: Jaguar | Version: 65786-26573960-Windows | Platform: Windows | 7.61 GiB)
 * The Jackbox Party Pack 2 (App name: Lilac | Version: 1.0.2 | Platform: Windows | 0.65 GiB)
 * The Jackbox Party Pack 3 (App name: Orchid | Version: 1.0.2 | Platform: Windows | 1.55 GiB)
 * The Jackbox Party Pack 4 (App name: Snowdrop | Version: 1.0.2 | Platform: Windows | 0.98 GiB)
 * The Jackbox Party Pack 5 (App name: Geranium | Version: 1.0.2 | Platform: Windows | 2.22 GiB)
 * The Jackbox Party Pack 6 (App name: Astrapia | Version: 1.1.0-win32 | Platform: Windows | 2.15 GiB)
 * The Jackbox Party Pack 7 (App name: 7fb6ce95c6d04a44a55fef16ea0db4c9 | Version: 1.0.1 | Platform: Windows | 1.51 GiB) * The Stanley Parable (App name: Freesia | Version: 1.0 | Platform: Windows | 2.07 GiB)
 * Unreal Engine (App name: UE_4.23 | Version: 4.23.1-9631420+++UE4+Release-4.23-Windows | Platform: Windows | 30.41 GiB)
 * Watch Dogs (App name: Jasper | Version: 1.0 | Platform: Windows | 14.07 GiB)
 * Watch Dogs 2 (App name: Angelonia | Version: 1.0 | Platform: Windows | 37.77 GiB)
 * while True: learn() (App name: 4f272a49a39742b795d63e1f483a7c7d | Version: 1.4.98.5144.39.win64 | Platform: Windows | 0.79 GiB)
 * World War Z (App name: Wombat | Version: 2.05 | Platform: Windows | 49.76 GiB)

Total: 56
```

**Comment by suchmememanyskill** [comment]
Hi, 

`Unexpected character encountered while parsing value` makes me think there is a JSON somewhere that has invalid or unexpected data. Would you mind [joining my dev discord](https://discord.gg/aH9rsuP) so you can send me some legendary configuration files?

**Comment by suchmememanyskill** [comment]
Fixed in d53ab09
